packages/grid/x-grid-modules/src/constants/eventsConstants.ts

```diff
@@ -1,4 +1,5 @@
 export const RESIZE = 'resize';
+export const ERROR = 'grid_error';
```

Comment:
I'm not sure to understand what's the convention with the variables, but this one seems to fit:
```suggestion
export const ERROR = 'error';
```

 I have noticed that, sometimes, there is an `_EVENT` suffix, sometimes not.

Reply:
`Error` is not possible as `EventEmitter` has a special process if you emit an error.

> I have noticed that, sometimes, there is an _EVENT suffix, sometimes not.

Yes I need to realign all that